### PR TITLE
Pos addons 12.0 pos network printer iot

### PR DESCRIPTION
**Pull request by KolushovAlexandr**

### CODE REVIEW

File: pos_printer_network_iot/doc/changelog.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+`1.0.0`
+-------
+
+- **Init version**
```
[the remark this answers is not in the document]
Comment (Ramil-Mukhametzyanov) [reply pos_printer_network_iot/doc/changelog.rst]: The IP-address field is not editable

**Review by Ramil-Mukhametzyanov — CHANGES_REQUESTED** [review]

**Review by Ramil-Mukhametzyanov — CHANGES_REQUESTED** [review]
Can not print on network order printer

image

image

image

**Review by Ramil-Mukhametzyanov — COMMENTED** [review]
http://192.168.1.126:8069/hw_proxy/status_network_printers?session_id=
Response:
{"result": [{"ip": "192.168.1.126", "status": "online", "name": "Printer"}, {"ip": "192.168.1.169", "status": "online", "name": "Kitchen Printer"}], "jsonrpc": "2.0", "id": 767669322}

**Comment by yelizariev** [comment]
The modules seems be merged to pos-addons

**Comment by yelizariev** [comment]
I'm wrong. IOT module is not published in pos-addons

**Pull request closed without merging**